src/game.cpp

```diff
@@ -6161,7 +6161,8 @@ void game::print_graffiti_info( const tripoint &lp, const catacurses::window &w_
 
     const int max_width = getmaxx( w_look ) - column - 2;
     if( m.has_graffiti_at( lp ) ) {
-        fold_and_print( w_look, ++line, column, max_width, c_light_gray, _( "Graffiti: %s" ),
+        fold_and_print( w_look, ++line, column, max_width, c_light_gray,
+                        m.ter( lp ) == t_grave_new ? _( "Graffiti: %s" ) : _( "Incription: %s" ),
```

Comment:
This causes "Inscription" label for all graffiti in look around view if terrain is not `t_grave_new` (i.e. walls, doors, etc).